robocop/checkers/spacing.py

```diff
@@ -42,7 +42,7 @@ def check_line(self, line, lineno):
             self.report("trailing-whitespace", lineno=lineno, col=len(stripped_line))
 
 
-class MissingTrailingBlankLineChecker(VisitorChecker):
+class EmptyLinesChecker(VisitorChecker):
```

Comment:
Figured out this name would be more suitable

Reply:
Sure it is!